### ISSUE TITLE
appendix/additional_resources - 1

### ISSUE THREAD

**Issue by RafaelFrance** [mixxxdj/manual#676]
2.4 manual
https://manual.mixxx.org/2.4/en/chapters/appendix/additional_resources
Change
**Mixxx Community Forums**: To search for answers, post your question or answer other DJ's questions, go to: `<https://mixxx.org/forums>`_
into
**Mixxx Community Forums**: To search for answers, post your question or answer other DJ's questions, go to: `Help & Support Forum <https://mixxx.discourse.group/c/support/6>`_

**Comment by ronso0** [comment]
Note that mixxx.org/forums i snot a dead end, redirect works.

**Comment by RafaelFrance** [comment]
Yes redirect works. Thanks.

**Comment by SamWhited** [comment]
I believe this can be closed, no? The redirect works and will presumably be worth keeping in case the forums ever move again in the future (in which case the redirect can be updated and the manual doesn't need to be).

**Comment by jrdevadattan** [comment]
I can update this link. I’ll replace the old `mixxx.org/forums` reference with the current Discourse Help & Support Forum URL.

**Comment by jrdevadattan** [comment]
I can update this link. I’ll replace the old `mixxx.org/forums` reference with the current Discourse Help & Support Forum URL.

**Comment by SamWhited** [comment]
@jrdevadattan Why? The old link works (and if the forums move again the redirect can be updated and continue to work). I don't think there's anything to do here.

**Comment by jrdevadattan** [comment]
You’re right, sorry. I missed the referenced commit from @mxmilkiib and should have checked that before commenting.

**Comment by ronso0** [comment]
> I believe this can be closed, no? The redirect works and will presumably be worth keeping in case the forums ever move again in the future (in which case the redirect can be updated and the manual doesn't need to be).

Yup,  agree.

If anyone wants to improve the manual there are other small tasks. @mxmilkiib created a meat/triage issue #924 which is worth looking at for a starter.
